website/views/shared/new-stuff.jade

```diff
@@ -17,11 +17,12 @@ h5 NEW ITEMS IN THE ENCHANTED ARMOIRE AND SEPTEMBER BACKGROUNDS!
       p.small.muted by Lemoness and SabreCat
       p.small.muted Art by Starsystemic and Kiwibot
 
-hr
-a(href='/static/old-news', target='_blank') Read older news
+hr.remove-on-static
```

Comment:
@crookedneighbor Is this actually the way I should hide those elements? Or is there some way to make sure they're not even on the page instead of just hidden?